amv/README.md

```diff
@@ -0,0 +1,22 @@
+# AMV vocabulary  
+AMV: Algorithm Metadata Vocabulary 
+
+# Algorithm Metadata Vocabulary (AMV), a vocabulary for capturing and storing the metadata about the algorithms (a procedure or a set of rules that is followed step-by-step to solve a problem, especially by a computer) and related objects, such as problem, software programmes. The snag faced by the researchers in the current time is the failure of getting relevant results when searching for algorithms in any search engine. AMV vocabulary is represented as a semantic model and produced as OWL file, which can be directly used by anyone interested to create and publish algorithm metadata as a knowledge graph, or to provide metadata service through SPARQL endpoint.   
+
+
+# Persistent URI 
+https://w3id.org/amv 
+
+
+## License
+This repository is licensed under Creative Commons 4.0 International. See the [LICENSE deed](LICENSE) for details.
```

Comment:
I'm not sure what the intent here is.  This file can't claim a license for the whole w3id repository.  Also the link will probably not redir to the correct location based on your htaccess rules.

Reply:
Sorry, again a typo issue. It supposes to be "AMV vocabulary." Blind copy/paste always is an issue. :-(

Reply:
BTW, thanks for your kind support.

Reply:
I have pushed the updated readme and .htaccess.